superbench/analyzer/data_analysis.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""A module for data analysis."""
+
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+from superbench.common.utils import logger
+
+
+def statistic(raw_data_df):
+    """Get the statistics of the raw data, including count, mean, std, min, max, 25%, 50%, 75%.
```

Comment:
I think 95th value is an important value, especially for latency analysis, could we add it?

Reply:
sure, added